### DISCUSSION TITLE
PostgREST Config File db-uri Password

### DISCUSSION THREAD

**Discussion by mchatrvd** [PostgREST/postgrest#3022] [Q&A] [answered]
Hi

What are the options to not hard code the password for the db-uri connection parameter in the PostgREST config file and have it read instead from somewhere else securely? Thanks in advance.

Regards,
Manish

**Comment by steve-chavez** [comment, marked as answer]
Hello,

You can get it from a file too:

```
db-uri = @filename
```

https://postgrest.org/en/stable/references/configuration.html#external-config-file

I believe the standard `pg_service.conf` file should work too

https://www.cybertec-postgresql.com/en/pg_service-conf-the-forgotten-config-file/

**Reply by mchatrvd** [reply]
Thanks so much for your help
